web/pages/api/dao/create.ts

```diff
@@ -17,11 +17,14 @@ async function postHandler(
   res: NextApiResponse<Dao | string>
 ) {
   try {
+    console.log('Got request to create DAO', req.body);
     const prisma = new PrismaClient();
     const result = await prisma.dao.create({
       data: {
         publicKey: req.body.publicKey,
         title: req.body.title,
+        about: req.body.about,
```

Comment:
REQ: update script `1-1-create-dao` to include new required info

Reply:
ok